src/v3/plugins/github/findReferences.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import * as N from "./nodes";
+import * as R from "./relationalView";
+import {parseReferences} from "./parseReferences";
+
+export type GithubReference = {|
+  src: N.TextContentAddress,
+  dst: N.ReferentAddress,
+|};
+
+type TextContentEntry =
+  | R.IssueEntry
+  | R.PullEntry
+  | R.CommentEntry
+  | R.ReviewEntry;
+export function* findReferences(
+  view: R.RelationalView
+): Iterator<GithubReference> {
+  function* allGithubEntities(): Iterator<R.Entry> {
+    yield* view.repos();
+    yield* view.issues();
+    yield* view.pulls();
+    yield* view.reviews();
+    yield* view.comments();
+    yield* view.userlikes();
+  }
+
+  const refToAddress: Map<string, N.StructuredAddress> = new Map();
+  for (const e of allGithubEntities()) {
+    const a = e.address;
+    refToAddress.set(e.url, a);
+    switch (e.type) {
+      case "USERLIKE":
+        refToAddress.set(`@${e.address.login}`, a);
+        break;
+      case "ISSUE":
+        refToAddress.set(`#${e.address.number}`, a);
+        break;
+      case "PULL":
+        refToAddress.set(`#${e.address.number}`, a);
+        break;
+      default:
```

Comment:
Consider adding here explicit cases for each other possible value of
`e.type`, and then using an `empty`-assertion in the `default`-case.
That way, when we add new GitHub node types, we’ll be forced to consider
this block, and either add a new reference type or explicitly decline to
do so.

Reply:
Done, in the new version.